channels/api.py

```diff
@@ -443,6 +443,12 @@ def update_post(self, post_id, text):
 
         return post.edit(text)
 
+    def pin_post(self, post_id, pinned):
+        """make the post sticky"""
+        post = self.get_post(post_id)
+        post.mod.sticky(pinned)
+        return post
```

Comment:
You can probably drop the return statement here since we don't need it

Reply:
right, that makes sense, will do